openstudiocore/src/model/Model.hpp

```diff
@@ -87,6 +88,9 @@ class MODEL_API Model : public openstudio::Workspace {
   /** @name Getters */
   //@{
 
+  /// Get the WorkflowJSON
```

Comment:
Yeah I merged in some other stuff as well, we could merge into os develop 2.0 to clean up this pr